modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,5 +1,6 @@
 @import 'configs/variables';
 
+@import 'components/dropdown';
```

Comment:
@albinojunior, could you import this CSS after the `.app-builder-root` because it's necessary to give priority to the components so that no utilities, pages, or configs will overwrite any dropdown CSS

Reply:
@interaminense, at this case I think that just move the import to after of config imports will prevent this. The others imports inside the `.app-builder-root` can't overwrite the dropdown styles, because the dropdown element is always created outside of the application components tree at DOM. Do you agree?

Reply:
Yes, I agree with this. Thanks, @albinojunior